krystal-visualization/src/main/java/com/flipkart/krystal/visualization/models/GraphGenerationResult.java

```diff
@@ -0,0 +1,11 @@
+package com.flipkart.krystal.visualization.models;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class GraphGenerationResult {
+  private final String html;
+  private final String fileName;
```

Comment:
Lets not put file name here. Let the caller decide what to do with the html
They might write to file, write to bloc store, or might return as is as a response in an API